### PR TITLE
Platform 1116 - adding Mercury App support for Universal Analytics

### PR DESCRIPTION
**Pull request by harnash**
Since Mercury has no simple way to query WikiFactory variables
for a specific wiki, new parameter is being sent with WikiVariables
response to allow enabling UA per wiki. It is temporary solution and
should be removed in next migration phases.

### CODE REVIEW

File: extensions/wikia/MercuryApi/models/MercuryApi.class.php [hunk 1]
```diff
@@ -125,6 +127,9 @@ public function getWikiVariables() {
 			'mainPageTitle' => Title::newMainPage()->getPrefixedDBkey(),
 			'theme' => SassUtil::getOasisSettings(),
 			'wikiCategories' => WikiFactoryHub::getInstance()->getWikiCategoryNames( $wgCityId ),
+			'analytics' => [
+				'UAEnabled' => $wgAnalyticsProviderUseUA,
+			],
```
Comment (macbre) [inline extensions/wikia/MercuryApi/models/MercuryApi.class.php]: `$wgAnalyticsProviderUseUA` does not have a default value defined. Please use `!empty($wgAnalyticsProviderUseUA)` instead.

Comment (harnash) [inline extensions/wikia/MercuryApi/models/MercuryApi.class.php]: Fixed in 28b58b0.

File: extensions/wikia/MercuryApi/models/MercuryApi.class.php [hunk 2]
```diff
@@ -125,6 +127,9 @@ public function getWikiVariables() {
 			'mainPageTitle' => Title::newMainPage()->getPrefixedDBkey(),
 			'theme' => SassUtil::getOasisSettings(),
 			'wikiCategories' => WikiFactoryHub::getInstance()->getWikiCategoryNames( $wgCityId ),
+			'analytics' => [
+				'UAEnabled' => !empty( $wgAnalyticsProviderUseUA ) ? $wgAnalyticsProviderUseUA : false,
```
Comment (macbre) [inline extensions/wikia/MercuryApi/models/MercuryApi.class.php]: `!empty` is enough, we need a boolean here, right?

Comment (harnash) [inline extensions/wikia/MercuryApi/models/MercuryApi.class.php]: Fixed in a7f6750.

**Comment by macbre** [comment]
LGTM

**Comment by harnash** [comment]
Merged

**Pull request closed without merging**